pkg/vault/contracts/VaultAdmin.sol

```diff
@@ -492,23 +508,12 @@ contract VaultAdmin is IVaultAdmin, VaultCommon, Authentication, VaultGuard {
             revert BufferSharesInvalidReceiver();
         }
 
-        uint256 newTotalSupply = _bufferTotalShares[wrappedToken] + amount;
-
-        _ensureMinimumTotalSupply(newTotalSupply);
```

Comment:
yeah I know this one is redundant, but it's just about being overly defensive and following the same pattern as with `mint` for BPT. I'd keep it symmetrical.

Reply:
Physically painful... but ok; I'll add a comment.